doc/whats_new/v0.23.rst

```diff
@@ -235,6 +235,10 @@ Changelog
   :class:`neural_network.MLPClassifier` by clipping the probabilities.
   :pr:`16117` by `Thomas Fan`_.
 
+- |Enhancement| Prevent the transformer from converting float32 to float64 in
```

Comment:
Maybe "Avoid converting float32 input to float64 in ..."?

Reply:
Much better ^^